src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck alfa = new Truck();
```

Comment:
```suggestion
        Machine alfa = new Truck();
```
нам потрібно використовувати абстрактний тип для оголошення посилання це нам дозволить використовувати поліморфізм.